app/src/main.ts

```diff
@@ -173,15 +173,21 @@ async function init(socket: WebSocket) {
     });
 })();
 
-for (
-  const signal of [
+
+let signals: Array<Deno.Signal>;
+if (Deno.build.os === "windows") {
+	signals = ['SIGINT', 'SIGBREAK'];
+} else {
+	signals = [
```

Comment:
nitpick: I think it might be more readable if you keep signals as a const and assign it with a ternary operator. 

Something like:
```ts
const win_signals = ['SIGINT', 'SIGBREAK'];
const unix_signals = ['SIGINT', 'SIGUSR2', '...etc'];
const signals = Deno.build.os === 'windows' ? win_signals : unix_signals;
```

This way it's immediately clear that there's a condition that dictates the value of signals when it's declared rather than it being assigned after the fact.

Reply:
Done. 🙏
I had to change the `for` loop declarations though.
Trying to assert as a `const` was throwing an assertion error, and trying to `as` with `Deno.Signal` was giving another error.
Assuming that's expected / required here since `signals` itself is variable instead of a constant literal now.

Reply:
Yeah, using let there should be fine. We've added `as const` to the individual arrays so we should still get type safety inside the loop as both `win_signals` and `unix_signals` satisfy the `Deno.Signal` type constraint. I'd expect to see an error in `Deno.addSignalListener` if it wasn't valid. Great work!